package.json

```diff
@@ -4,20 +4,18 @@
     "private": true,
     "scripts": {
         "start": "npm run serve",
```

Comment:
While trying to debug the "preview" timeout problem I realized that `npm start` is broken.